daemon/resources/razer.conf

```diff
@@ -10,6 +10,14 @@ sync_effects_enabled = True
 # Turn off the devices when the systems screensaver kicks in
 devices_off_on_screensaver = True
 
+# Mouse DPI notifier
+mouse_dpi_notifier = False
+
+# Mouse DPI notification frequency [ms] (0 to disable)
+# WARNING: The openrazer-daemon will poll an openrazer DBUS service at this frequency
```

Comment:
"[...] will poll the mouse at this frequency"

Also, why have both `= 0` here to disable but also the boolean? Can we get rid of this boolean for example?